### DISCUSSION TITLE
Disable Add button

### DISCUSSION THREAD

**Discussion by anhuet** [firecmsco/firecms#288] [Q&A]
I have an array of items, I want the user  not be able to add items in the CMS by disable, hide ... this "Add" button, is it possible?

**Comment by fgatti675** [comment]
Hi @anhuet 
You can set the `create` flag to false under `permissions` at the collection level

**Reply by anhuet** [reply]
Hi @fgatti675, thanks for reply
So is it possible with children field of this collection?

**Reply by fgatti675** [reply]
Ah sorry, I didn't understand it was only within an array, you can discard my previous comment, which applies to collections.
This is not possible at this time unfortunately!

**Reply by anhuet** [reply]
tks @fgatti675
